docker-compose.yaml

```diff
@@ -1,21 +1,29 @@
 version: '3.7'
```

Comment:
So you can use Docker Compose file directly?

Reply:
I used the `docker-compose.yaml` file to get everything running before translating to K8s. Once everything was running I used `kompose` on the `docker-compose.yaml` file to generate the base manifests for K8s. Then I went through them by hand and edited them to remove the extra bits we don't need. Pretty quick process given the number of services there are